.github/workflows/ci.yml

```diff
@@ -110,8 +110,15 @@ jobs:
 
                 curl -L -O https://tiker.net/ci-support-v0
                 . ci-support-v0
-                ./configure.py --cl-use-shipped-ext
-                build_py_project_in_conda_env
+
+                install_conda_deps
+
+                export CL_USE_SHIPPED_EXT=on
+                export CL_INC_DIR="$CONDA_PREFIX/Library/include"
+                export CL_LIB_DIR="$CONDA_PREFIX/Library/lib"
```

Comment:
Why does this still require exporting the variables here? Does cmake not find them? Might be worth adding a note somewhere if this is actual issue.

Reply:
The problem is, I don't know what the OpenCL library is called on Windows, so I had to revert the commit that added conda detection on Windows: https://github.com/inducer/pyopencl/pull/755/commits/e4906a63548e646c40e20b61df0f7319568ea606

Reply:
found it :-) [96814a2](https://github.com/inducer/pyopencl/pull/755/commits/96814a222df3bec7ceee1d52d9c1199a8690f8ef)